### PR TITLE
add call to remove land locked cells

### PR DESCRIPTION
**Pull request by mark-petersen**
This is needed to create a V3 mesh with COMPASS.  It has a matching PR in MPAS-Tools.

### CODE REVIEW

**Comment by mark-petersen** [comment]
See https://github.com/MPAS-Dev/MPAS-Tools/pull/220 for matching PR.

File: testing_and_setup/compass/general.config.ocean [hunk 1]
```diff
@@ -47,4 +46,5 @@ mesh_generator = FULL_PATH_TO_EXECUTABLE_MESH_GENERATION_SCRIPT
 mesh_database = FULL_PATH_TO_LOCAL_MESH_DATABASE
 initial_condition_database = FULL_PATH_TO_LOCAL_INITIAL_CONDITION_DATABASE
 geometric_features = FULL_PATH_TO_LOCAL_CHECKOUT_OF_GEOMETRIC_FEATURES_DATABASE
+mpas_tools = FULL_PATH_TO_MPAS_TOOLS_REPO
```
[the remark this answers is not in the document]
Comment (mark-petersen) [reply testing_and_setup/compass/general.config.ocean]: I thought about that too.  It depends on whether people use different executables for gnu/intel debug/optimized for testing.

Regardless, we are reorganizing the directories in MPAS-Tools, I'd like to save that for a PR after the reorg.

**Comment by xylar** [comment]
Thanks for the updated version.  I'm retesting now.

**Comment by xylar** [comment]
@mark-petersen, for the sake of documentation, could you describe your last 2 commits?  Namely, are critical passages widened for  the v3 meshes we're already using?  Or would this be something you suggest we do going forward?

**Comment by mark-petersen** [comment]
Good point.  So far on this PR I am able to confirm I can reproduce the correct "V3" number of grid cells for:
- EC60to30 235160 *requires critical passages*
- EC60to30wLI 236358 *requires critical passages*
- RRS30to10 1445361 *requires critical passages*

So RRS30to10 requires `--with_critical_passages` otherwise the Baltic Sea disappears.  I'm building the RRS30to10wLI right now, with and without critical passages, to confirm which is correct compared to the the current uploaded file.

**Comment by mark-petersen** [comment]
Also confirmed this matches uploaded grid:
- RRS30to10wLI 1460217 *requires critical passages*

**Pull request merged**